removeFromArray/removeFromArray.js

```diff
@@ -1,9 +1,7 @@
 // we have 2 solutions here, an easier one and a more advanced one.
 // The easiest way to get an array of all of the arguments that are passed to a function
```

Comment:
Actually just a quick fix to make, let's update this to:

```suggestion
// The easiest way to get an array of the rest of the arguments that are passed to a function
```